### ISSUE TITLE
The 0.4.4 version

### ISSUE THREAD

**Issue by stevepdelf** [Atomistica/atomistica#9]
I notice in the download of 0.4.4 after properly unzipped does not have file structure as outlined in the instructions.  For example the 'build_lammps' directory is absent.  Is the 0.4.4 to be placed on top of the 0.4.3?  Or are their new install steps coming?

**Comment by pastewka** [comment]
The zip should be self-contained. Where did you get the zip from, pypi or the green download button on GitHub? (The pypi zip contains just the Python interface.)

**Comment by pastewka** [comment]
I just checked the github download, it did contain the "build_lammps" folder. Can you give more details on where you obtained the zip file?

**Comment by stevepdelf** [comment]
I thought i downloaded it from https://lammps.sandia.gov/.  I can not verify this morning because the site is down.  Looking at what I downloaded though it definitely looks like I downloaded the Python interface.  Being that I am subject to the human condition of fallability I most likely downloaded something different than I intended.  I'll look for it here on github and download it and see where that takes me. 

"I offer a complete and utter retraction. The imputation was totally without basis in fact, and was in no way fair comment, and was motivated purely by malice, and I deeply regret any distress that my comments may have caused you, or your family, and I hereby undertake not to repeat any such slander at any time in the future."
